### PR TITLE
[jarvis] - script.maps.browser 1.0.0 + ExtendedInfo Update

### PR DESCRIPTION
**Pull request by phil65**

### CODE REVIEW

**Comment by ronie** [comment]
if Maps Browser is supposed to be a script as well as a python module,
you should split them into separate addons.

for python modules (though i'm not an expert here) i think all *.py files need to be in a subfolder in /lib.

also the name of that folder needs to be unique so it doesn't clash with names of other python modules.

so /lib/googlemaps/*.py should work.

for example, have a look at: https://github.com/xbmc/repo-scripts/tree/gotham/script.module.chardet

**Comment by phil65** [comment]
Sorry, that extension point was an old leftover and not used (yet). I removed it now, thx.

File: script.maps.browser/resources/language/English/strings.po [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+msgctxt "Addon Summary"
+msgid "Google Maps Browser (including Street View)"
+msgstr ""
+
+msgctxt "Addon Description"
+msgid ""
+"This Maps Browser Add-on allows you to explore the map with a remote. It can also show Places (Source: FourSquare), "
+"events (source: Eventful), geotagged images and much more on the map. Also offers skin integration."
```
Comment (ronie) [inline script.maps.browser/resources/language/English/strings.po]: not sure if it's valid like this, but perhaps stick with using a single line:
see: https://github.com/xbmc/xbmc/commit/5e7507e663461e023aec4a958a6d6dadaea493b9

**Comment by ronie** [comment]
as mention in the inline comments, no idea if this will work:
script.maps.browser/resources/skins/Default/color/defaults.xml

these shouldn't be need afaik:
script.maps.browser/__init__.py
script.maps.browser/resources/__init__.py

icon.png needs to be 256x256px

**Comment by phil65** [comment]
the __ init __ in resources is needed i think because i dont use sys.path.append(), but import from the subfolders directly. The other one is superfluous, I removed it.
The multiline string in .po file is valid, I might ask alan why his system translates those to a single-line string.
The rest of your comments was addressed and fixed.

**Comment by phil65** [comment]
I think we should also allow up to 512x512 for add-ons nowadays. Agreed?

**Comment by kib** [comment]
Makes sense to me

**Comment by ronie** [comment]
not my call, i received some slack when i (not that long ago) told people it was ok to use a 1920x1080 fanart.jpg instead of 1280x720.

the only downside i can think of is it's going to take ~4 times as long to fetch all the thumbs the first time you enter the addon browser.
if skins use a 3x5 panel layout, kodi will need to download 15 512x512px images at once.

**Comment by phil65** [comment]
isnt that the same (or even worse) with movie posters?
I am also not sure if png filesize correlates with pixel amount that much. ;)

**Comment by phil65** [comment]
good to go now?

**Pull request merged**